### ISSUE TITLE
Drop net45* and net461 support. Move to min of 462

### ISSUE THREAD

**Issue by SimonCropp** [microsoft/vstest#3755]
[.NET Framework 4.5.2, 4.6, 4.6.1 reached End of Support on April 26, 2022](https://devblogs.microsoft.com/dotnet/net-framework-4-5-2-4-6-4-6-1-will-reach-end-of-support-on-april-26-2022/) 

Issues this will likely help with 

 * https://github.com/microsoft/vstest/issues/2316
 * https://github.com/microsoft/vstest/issues/2488
 * https://github.com/microsoft/vstest/issues/3733

**Comment by nohwnd** [comment]
I agree, it is in the works and we are planning that for 17.4. 

- Move to netcoreapp3.1, net462 and netstandard2.0.
- We will (most likely) keep the old targets for ObjectModel and it's dependencies, so adapters don't have to update immediately. I know that XUnit already did.

1. We are investigating how to get rid of both newtonsoft.json and nuget.frameworks in testhost, because those are our only external depencencies. 
2. System.Text.Json probably won't work for us because it is not in .NET Framework and does not use the standard data annotations, which would force us to depend on it for object model. Making the Newtonsoft.Json problem worse, not better.
3. I had a PR for that, which never got merged, and when I closed it because we never got to merge it and there were no new reports of the issue, this issue pops up again, 30 minutes or so after I closed it.
